src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comment = [] }) => (
+  <div className="CommentList">
+    {comment.map(commentary => (
+      commentary === []
```

Comment:
Consider using commentary length here. Object comparison is a comparison by link. In this case, the result would be always false